src/content/1.7/modules/concepts/hooks/list-of-hooks.md

```diff
@@ -1410,6 +1573,11 @@ displayBackOfficeHeader
     
 displayBackOfficeOrderActions
 : 
+    **(deprecated since 1.7.7)**
+    → `actionGetAdminOrderButtons`
+
+    This hook displays content in the order view page after action buttons (or aliased to side column in migrated page)
```

Comment:
> (or aliased to side column in migrated page)

I honestly don't understand this part 😅

Reply:
This one is a bit special:
- since 1.7.7 it is aliased to `displayAdminOrderSide` to avoid losing all the former actions, but it is not displayed in the same part on the page
- but the recommended way is to use the new `actionGetAdminOrderButtons` which is cleaner and adds action buttons at the top o the view

The alias feature was decided to keep a bit a backward retro compatibility but still motivate developers to migrate their module if they want their buttons to be on top of the page I'm not much of a fan of this, I think purely replace the hook would be much more efficient to motivate them ^^ (as it was done on other hooks)

Reply:
I think that part must be clarified in the text.